### PR TITLE
Features/crew info

### PR DESCRIPTION
**Pull request by cguardia**

### CODE REVIEW

File: src/vms/static/app/workparty.js [hunk 1]
```diff
@@ -313,6 +313,10 @@ var WorkPartyCrewCorner = module.exports.WorkPartyCrewCorner = React.createClass
 					{volunteer.mine && <td><VolunteerCancelButton value={{cancelReg: volunteer.id}} 
 																  action={vol_cancel_action}
 																  volUnRegHandler={cancelVol} /></td>}
+                    <td><span className={volunteer.waiver_signed ? 'icon-ok' : 'icon-cancel'}></span></td>
+                    <td>{volunteer.under_18 ? <span className='icon-ok'></span> : <span>N/R</span>}</td>
+                    {medReq && <td><span className='icon-ok'></span></td>}
+                    <td><span className='icon-ok'></span></td>
```
Comment (davisagli) [inline src/vms/static/app/workparty.js]: These are meant to show a checkbox if the contact has completed the form. It looks like it's currently always showing a checkbox for the parental release, medical form and backcountry experience columns.

Comment (davisagli) [inline src/vms/static/app/workparty.js]: Also, I'm guessing we should only show the last 3 columns if those particular documents are required for this work party.

Comment (cguardia) [inline src/vms/static/app/workparty.js]: I couldn't find any way to get info about those forms being filled in. If you know that, I can make the change right away.

Comment (davisagli) [inline src/vms/static/app/workparty.js]: We probably need to discuss that with Matthew. For medical and backcountry it's a matter of whether there's an completed form for the contact from the past year, which isn't an easy thing to query for efficiently. But maybe we can add rollup fields on the contact. We haven't done any work on parental releases yet.

Comment (cguardia) [inline src/vms/static/app/workparty.js]: I assumed there was work pending on those things, which is why I just put the ok icon. I was going to comment about that in the testing instructions.

Comment (davisagli) [inline src/vms/static/app/workparty.js]: Okay I'll merge this and we can adjust once the rollup fields are in place.

**Pull request merged**